src/odemis/gui/comp/miccanvas.py

```diff
@@ -1165,6 +1166,110 @@ def update_drawing(self):
 
         if self.view:
             self.update_thumbnail()
+            
+class NavigableBarPlotCanvas(BarPlotCanvas):
+    
+    def __init__(self, *args, **kwargs):
+        super(NavigableBarPlotCanvas, self).__init__(*args, **kwargs)
+        self.abilities |= {CAN_DRAG}
+        
+        # When resizing, margin to put around the current size
+        # TODO: Maybe make the margin related to the canvas size?
+        self.default_margin = 512
+        self.margins = (self.default_margin, self.default_margin)
+
+        # the position the view is asking to the next buffer recomputation
+        # in buffer-coordinates: = 1px at scale = 1
```

Comment:
What is this comment about? Probably not `data_buffer`. Is that `window_x/y`? => Move to the right place, or make it more explicit in the comments.